ros/cpp/src/cpp_pubsub/src/publisher_member_function.cpp

```diff
@@ -87,8 +100,8 @@ class MinimalPublisher : public rclcpp::Node
 
   private:
     size_t count_;
-    std::string server_ip_addr = "169.254.103.120";
-    
+    std::string server_ip_addr = "192.167.1.90";

```

Comment:
OK for now but we need to make this a node parameter in the future. https://docs.ros.org/en/foxy/Tutorials/Using-Parameters-In-A-Class-CPP.html#